challenges/objects-arrays.js

```diff
@@ -98,28 +240,36 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 */
 
 const lowerCase = [];
-console.log(lowerCase); 
+
+zooAnimals.map(animal => lowerCase.push(animal.animal_name.toLowerCase()));
+console.log(lowerCase);
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
 const lowerPopulation = [];
+largerPopulation = zooAnimals.filter(animal =>
```

Comment:
Like map, filter creates its own new array. You don't need to push to an empty array.
```suggestion
lowerPopulation = zooAnimals.filter(animal => animal.population < 5)
```